intro/models.md

```diff
@@ -151,21 +151,24 @@ As you can see in the example, due to the different column types, the values can
 | `formula`                                      | `string`                                                                               | `"Albert Summer"`                                                                                                                                                     | Contains only the result of the formula.                                                                                                                |
+| `auto-number`                                  | `string`                                                                               | `"0001"`                                                                                                                                                              |                                                                                                                                                         |
 | `button`                                       | `null`                                                                                 | `null`                                                                                                                                                                | Always `null`                                                                                                                                           |
-| `link`                                         | `array`                                                                                | `[{"row_id": "Mr-3QnQ5Q-eeEqFBevtOkw", "display_value": "Google News"}]`                                                                                              |                                                                                                                                                         |
+| `link`                                         | `array`                                                                                | `["Mr-3QnQ5Q-eeEqFBevtOkw"]`                                                                                                                                          | Array of row IDs.                                                                                                                                       |
 | `link-formula`                                 | `array`, `string` or `integer`                                                         | `["Todo 1", "Todo 2"]`, `Open Pending`, `2`                                                                                                                           | The output depends on the link formula type. Lookup returns an array, rollup returns a string and findmax returns an integer.                           |
```

Comment:
I checked all link-formula types and the output differs wildly:

| Type | /api-gateway | /dtable-server |
|--------|--------|--------|
| lookup | `array<string>` | `string` (CSV) |
| countlinks | `integer` | `string` (numeric) |
| rollup | `integer` | `string` (numeric) |
| findmax/findmin | `array<integer>` | `string` (numeric) | 

(tested using 4.4.10 @ stage)